### PR TITLE
Allow multiple band intensity symbologies

### PR DESCRIPTION
**Pull request by jterry64**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [X] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [X] Check the commit's or even all commits' message styles matches our requested structure.
- [X] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?
`year_intensity` symbology option only works with single band rasters, and the name also implies it works only on years, even though it works on any single band integer raster. 

Issue Number: GTC-2296

## What is the new behavior?

Refactor `year_intensity` to `value_intensity` symbology which generically will create intensity symbology from 1 or 2 band rasters.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py [hunk 1]
```diff
@@ -217,10 +217,10 @@ async def raster_tile_cache_validator(
     symbology_type = input_data["creation_options"].get("symbology", {}).get("type")
     if symbology_type:
         symbology_info = symbology_constructor[symbology_type]
-        req_input_bands: Optional[int] = symbology_info.req_input_bands
+        req_input_bands: Optional[List[int]] = symbology_info.req_input_bands
 
         if req_input_bands and (
-            req_input_bands != source_asset.creation_options.get("band_count", 1)
+            req_input_bands in source_asset.creation_options.get("band_count", [1])
```
Comment (solomon-negusse) [inline app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py:223]: When the `int` `band_count` is passed, don't we have to transform it to the list of band ids here?

Comment (solomon-negusse) [reply app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py:223]: Also, will the `SymbologyInfo` `req_input_bands` arguments have to be updated to lists here: https://github.com/wri/gfw-data-api/blob/master/app/tasks/raster_tile_cache_assets/symbology.py#L645?

Comment (jterry64) [reply app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py]: @solomon-negusse whoops, I got the check backwards! Fixed

**Comment by jterry64** [comment]
@solomon-negusse @dmannarino I fixed tests and comments, any more thoughts on this PR or can I merge it?

**Review by solomon-negusse — APPROVED** [review]

**Review by dmannarino — APPROVED** [review]
I apologize for taking so long to review!
I initially felt like this was making year_intensity_symbology too complicated, but on after thinking about it a bit I guess it really is just extending it in a sensible way to 2 source bands. So 👍!

**Pull request merged**